topostats/processing.py

```diff
@@ -266,13 +266,16 @@ def process_scan(
                             LOGGER.info(f"[{filename}] : *** DNA Tracing ***")
                             tracing_stats = defaultdict()
                             for direction, _ in grainstats.items():
-                                tracing_stats[direction] = trace_image(
+                                tracing_results = trace_image(
                                     image=filtered_image.images["gaussian_filtered"],
                                     grains_mask=grains.directions[direction]["labelled_regions_02"],
                                     filename=filename,
                                     pixel_to_nm_scaling=pixel_to_nm_scaling,
                                     **dnatracing_config,
                                 )
+                                tracing_stats[direction] = tracing_results["statistics"]
+                                # ordered_traces = tracing_results["ordered_traces"]
+                                # image_trace = tracing_results["image_trace"]
```

Comment:
Commented out code accidentally left in, or is this courteously hinting at how I can add plotting after this PR? 😄

Reply:
I left that in on purpose, figured you would be returning these and need them but `pylint` complained that they were assigned but not used so I just `#` them.